flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -385,11 +398,33 @@ class SentryFlutterOptions extends SentryOptions {
 class _SentryFlutterExperimentalOptions {
   /// Replay recording configuration.
   final replay = SentryReplayOptions();
-}
 
-/// Callback being executed in [ScreenshotEventProcessor], deciding if a
-/// screenshot should be recorded and attached.
-typedef BeforeScreenshotCallback = FutureOr<bool> Function(
-  SentryEvent event, {
-  Hint? hint,
-});
+  /// Privacy configuration for masking sensitive data in the Screenshot and Session Replay.
```

Comment:
I know I've originally written this in the other PR but that doesn't make it correct :D
```suggestion
  /// Privacy configuration for masking sensitive data in screenshots and Session Replay.
```